protocol/src/main/java/org/threadly/litesockets/protocols/http/shared/HTTPUtils.java

```diff
@@ -1,14 +1,40 @@
-package org.threadly.litesockets.protocol.http.structures;
+package org.threadly.litesockets.protocols.http.shared;
 
 import java.nio.ByteBuffer;
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
 
-import org.threadly.litesockets.protocol.http.structures.HTTPConstants.REQUEST_TYPE;
+import org.threadly.litesockets.utils.MergedByteBuffers;
 
+/**
+ * 
+ * 
+ * @author lwahlmeier
+ *
+ */
 public class HTTPUtils {
-  public static final HTTPHeaders DEFAULT_HEADERS= new HTTPHeaders(HTTPConstants.DEFAULT_HEADERS_MAP);
-  public static final HTTPRequestHeader DEFAULT_REQUEST_HEADER= new HTTPRequestHeader(REQUEST_TYPE.GET, "/", null, HTTPConstants.HTTP_VERSION_1_1);
+  
+  public static String bbToString(ByteBuffer bb) {
+    byte[] ba = new byte[bb.remaining()];
+    bb.get(ba);
+    return new String(ba);
+  }
+  
+  public static int getNextChunkLength(final ByteBuffer bb) {
+    MergedByteBuffers mbb = new MergedByteBuffers();
+    mbb.add(bb);
+    int pos = mbb.indexOf(HTTPConstants.HTTP_NEWLINE_DELIMINATOR);
+    try {
+      if(pos >= 0) {
+        String csize = mbb.getAsString(pos);
+        return Integer.parseInt(csize, HTTPConstants.HEX_SIZE);
+      }
+    } catch(Exception e) {
```

Comment:
yep, though was I didnt care what the error was, but that should be the only one possible afaik.